onnx/common/ir.h

```diff
@@ -837,6 +837,22 @@ friend struct Value;
     initializers_.push_back(std::move(initializer));
     initializer_names_.push_back(std::move(name));
   }
+  void eraseInitializer(std::string name) {
+    initializers_.erase(
+        std::remove_if(
+            initializers_.begin(),
+            initializers_.end(),
+            [&name](Tensor initializer) {
```

Comment:
Tensor& initializer? otherwise the cost may be very high if initializer is large.